docs/_config.yml

```diff
@@ -9,7 +9,9 @@ description: >- # this means to ignore newlines until "baseurl:"
   documentation on how to use ICU.
 
 baseurl: "/icu" # the subpath for the site
-url: "https://unicode-org.github.io" # the base hostname & protocol for your site
+# url: "https://unicode-org.github.io" # the base hostname & protocol for your site
+# DO NOT MERGE: docs demo override, revert changes to this file before merging PR:
```

Comment:
I usually forget to remove the url override before merging, so what I started doing is leave this setting alone in the PR branch, and then I maintain a "demo branch" that rebases [a single commit](https://github.com/echeran/icu/commit/1c1b0427177d212fcf4c49504959b2543d0ea2bb) on top of the PR branch. And then I configure GH pages to always point to the demo branch. It comes at the cost of remembering that every push to the PR branch requires a rebase and push of the demo branch, too.

You may find that this process is more effort than it's worth, so do whatever works best for you.

Reply:
Thanks for the tip! I'll consider my options and see what works best for me!